src/index.html

```diff
@@ -107,6 +107,7 @@
     <div id="app"></div>
 
     <script src="https://unpkg.com/leaflet@1.0.1/dist/leaflet.js"></script>
+    <script src="/javascripts/PruneCluster.js"></script>
```

Comment:
Where should we put this? What do you think @oscarferrandiz? I tried to use ´npm install prunecluster´ but I could't access to its functions...